docs/examples/Transformer Examples.ipynb

```diff
@@ -40,7 +40,8 @@
     "import numpy as np\n",
     "import warnings\n",
     "\n",
-    "with warnings.catch_warnings(action=\"ignore\", category=DeprecationWarning):\n",
+    "with warnings.catch_warnings():\n",
+    "    warnings.simplefilter(\"ignore\", category=DeprecationWarning)\n",
```

Comment:
any specific reason for this change? not an issue, just interested cause i didn't expect it here

Reply:
Old way threw error at me

Reply:
Possibly cause you are using python 3.10. Although we do support pyhon 3.10, i am not sure whether we should require 3.10 in the examples. We also want to promote best practices like live at head, so requiring python 3.12 makes sense from that point of view. I guess that's out of scope of this PR so we can discuss offline